agora_analytics/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "hse7fl02-376a-33d0-7lpf-18s3r1284632",
+  "app_id": "agora-analytics",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View Agora Analytics Collector data in Metrics Explorer",
+    "title": "The Agora Analytics",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
These two images are very similar. Could you please upload a screenshot of your UI, or of the Datadog dashboard associated with this integration?

Also, images that are not 16:9 aspect ratio will get cut off in the media carousel preview.

Reply:
removed one of the similar image and added a screentshot of our UI

Reply:
@zjuzpz We typically like the images to show the value of the integration like graphs, dashboards, etc. The first image can be removed in this case. Do you have any images of graphs or dashboards?

Reply:
Additionally, the second image is difficult to understand what is being shown. Can you provide a title on the graph for more insight?

Reply:
@emarsha94 The first image has been removed, and we do not have images of graphs or dashboards. 
For the second image, the "avg:agora.rtc.app_id.audio_freeze_rate{*}" is a kind of the title of the graph, where rtc is an agora product, app_id is something like user_id, audio_freeze_rate is the metric name related to the rtc product. I am not sure what kind of title you are asking (if a title like "audio freeze rate" be fine?)

Reply:
It looks like the screenshot of the UI was removed - I think that would be helpful to paint a picture in customer's heads of what the data being sent in looks like. Not blocking, but we'd recommend adding another image here.

Reply:
> @zjuzpz We typically like the images to show the value of the integration like graphs, dashboards, etc. The first image can be removed in this case. Do you have any images of graphs or dashboards?
@anawishnoff  Hi, ana, I removed the screenshot of the UI and uploaded a dashboard snapshot as Emily asked.